pkgs/by-name/ca/cargo-ament-build/package.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "cargo-ament-build";
+  version = "0.1.9";
+
+  src = fetchFromGitHub {
+    owner = "ros2-rust";
+    repo = "cargo-ament-build";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-5D0eB3GCQLgVYuYkHMTkboruiYSAaWy3qZjF/hVpRP0=";
+  };
+
+  cargoLock.lockFile = ./Cargo.lock;
+
+  postPatch = ''
+    ln -s ${./Cargo.lock} Cargo.lock
+  '';
+
+  nativeBuildInputs = [ pkg-config ];
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "This cargo plugin is a wrapper around cargo build which installs build artifacts in a layout expected by ament and ROS 2 tools.";
+    homepage = "https://github.com/Guelakais/cargo-ament-build/tree/main";
```

this is the wrong homepage
```suggestion
    homepage = "https://github.com/ros2-rust/cargo-ament-build/";
```

---

done